distributed/core.py

```diff
@@ -462,9 +462,6 @@ def send_recv_from_rpc(**kwargs):
             raise gen.Return(result)
         return send_recv_from_rpc
 
-    def __del__(self):
-        self.close_streams()
```

Comment:
Well, looking at the Tornado source code, IO streams actually don't seem to feature a custom garbage collection mechanism, so perhaps `__del__` is really needed. I'll take a look.